programmers/난이도별/level01.문자열_내_p와_y의_개수/rockmiin.py

```diff
@@ -0,0 +1,11 @@
+def solution(s):
+    cnt_p=0; cnt_y=0
+    for i in s:
+        if i == 'P' or i=='p':
+            cnt_p+=1
+        if i == 'Y' or i == 'y':
+            cnt_y+=1
+    if cnt_p==cnt_y: return True
+    return False
```

Comment:
```suggestion
    for i in s:
        if i in ['P', 'p']:
            cnt_p+=1
        if i in ['Y', 'y']:
            cnt_y+=1
    return cnt_p == cnt_y
```
Boolean의 경우 바로 return 해도 된답니다!

Reply:
감사합니다 ! :)